tests/__mocks__/@myparcel/js-sdk/dist/data/fakeCarriersResponse.js

```diff
@@ -17,6 +17,15 @@ const allCarrierData = [
       logo_png: '/skin/general-images/carrier-logos/bpost.png',
     },
   },
+  {
+    id: 3,
+    name: 'cheapcargo',
+    human: 'Cheap Cargo',
+    meta: {
+      logo_svg: '/skin/general-images/carrier-logos/svg/cheapcargo.svg',
```

Comment:
persoonlijk had ik deze een extra laagje gegeven.

```logo: { svg: '/skin/general-images/carrier-logos/svg/cheapcargo.svg' } }```

food for thought.

Reply:
Cool, maar dit is een response van https://api.myparcel.nl/carriers :P

Reply:
dan houdt dat op :)